### PR TITLE
fix(tablets): remove tablets from experimental features

### PR DESCRIPTION
**Pull request by yarongilor**
fix(tablets): remove tablets and consistent-topology-changes from experimental features
    
            tablets feature is not experimental anymore and is default
            to be enabled. tablets is dependent on consistent-topology-changes.
            refs: https://github.com/scylladb/scylladb/pull/18898
refs: https://github.com/scylladb/scylladb/pull/18898

i'm not sure this PR is needed and about its dependencies or anything else.. 
The PR also assumes there's no need of updating scylla yaml with: `"enable_tablets": "false"`
### Testing

- [ ]

### PR pre-checks (self review)

- [ ] I added the relevant `backport` labels
- [ ] I didn't leave commented-out/debugging code

### Reminders

- Add New configuration option and document them (in `sdcm/sct_config.py`)
- Add unit tests to cover my changes (under `unit-test/` folder)
- Update the Readme/doc folder relevant to this change (if needed)

### CODE REVIEW

File: defaults/test_default.yaml [hunk 1]
```diff
@@ -136,7 +136,6 @@ scylla_version: ''
 test_upgrade_from_installed_3_1_0: false
 target_upgrade_version: ''
 disable_raft: true
```
Comment (bhalevy) [inline defaults/test_default.yaml:138]: Is this still true?  Do we disable raft by default?

File: upgrade_test.py [hunk 2]
```diff
@@ -371,15 +368,6 @@ def _rollback_node(self, node, upgrade_sstables=True):
         node.run_nodetool("snapshot")
         node.stop_scylla_server(verify_down=False)
 
-        if self.params.get("enable_tablets_on_upgrade"):
-            with node.remote_scylla_yaml() as scylla_yml:
-                current_experimental_features = scylla_yml.experimental_features
-                current_experimental_features.remove("tablets")
```
Comment (bhalevy) [inline upgrade_test.py:377]: Should we instead edit scylla.yaml to remove the newly introduced `enable_tablets` config option?
(See scylladb/scylladb#18898)

File: defaults/test_default.yaml (hunk 1 above)
Comment (yarongilor) [reply defaults/test_default.yaml:138]: IIUC this flag is not relevant anymore, unless still relevant to only 1 out of 2 raft features. @aleksbykov , @roydahan , can you please confirm?

Comment (roydahan) [reply defaults/test_default.yaml:138]: Probably not.
@aleksbykov / @temichus ?

**Comment by roydahan** [comment]
@yarongilor your PR shouldn't touch raft experimntal - @aleksbykov / @temichus please organise it so whatever should be tested with "raft disabled" (consistent-topology) is correctly configured and remove all the unnecessary flags / configurations.

@yarongilor your PR should flip the flag, now to support "disabling" tablets and have the duplicated jobs that are testing tablets with "disabled raft" - @aleksbykov & @temichus should be synced because some of these should be for 6.0 (so maybe they already working on it).

**Comment by yarongilor** [comment]
> @yarongilor your PR shouldn't touch raft experimntal - @aleksbykov / @temichus please organise it so whatever should be tested with "raft disabled" (consistent-topology) is correctly configured and remove all the unnecessary flags / configurations.
> 
> @yarongilor your PR should flip the flag, now to support "disabling" tablets and have the duplicated jobs that are testing tablets with "disabled raft" - @aleksbykov & @temichus should be synced because some of these should be for 6.0 (so maybe they already working on it).

Conclusions:
This PR will only change tablets flag and will modify a directory to have tablets disabled. 
The upgrade will also be adjusted to add the tablets flag.

File: defaults/test_default.yaml (hunk 1 above)
Comment (aleksbykov) [reply defaults/test_default.yaml:138]: This flag is only was used for rolling upgrade tests. Job with disabled raft uses additional yaml with relevant

**Comment by yarongilor** [comment]
Addressed in https://github.com/scylladb/scylla-cluster-tests/pull/7295 and https://github.com/scylladb/scylla-cluster-tests/pull/7655

**Pull request closed without merging**